independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/ConfiguratorBase.java

```diff
@@ -31,6 +32,7 @@ public THIS param(String name, boolean value) {
     }
 
     public THIS param(String name, boolean[] value) {
+        Objects.requireNonNull(value, "Parameter value can't be null");
```

Comment:
I'd presonally use something like `params.put(name, Objects.requireNonNull(value));` becaue the message does not seem to be very helpful...

Reply:
Updated the PR.